contracts/sophia.md

```diff
@@ -814,9 +840,16 @@ private public rec stateful switch true type record datatype
 - `QCon = (Con\.)+Con` qualified constructor
 - `TVar = 'Id` type variable (e.g `'a`, `'b`)
 - `Int = [0-9]+|0x[0-9A-Fa-f]+` integer literal
-- `Hash = #[0-9A-Fa-f]+` hash literal
+- `Bytes = #[0-9A-Fa-f]+` byte array literal
 - `String` string literal enclosed in `"` with escape character `\`
 - `Char` character literal enclosed in `'` with escape character `\`
+- `AccountAddress` base58-encoded 32 byte account pubkey with `ak_` prefix
+- `ContractAddress` base58-encoded 32 byte contract address with `ct_` prefix
+- `OracleAddress` base58-encoded 32 byte oracle address with `ok_` prefix
+- `OracleQueryId` base58-encoded 32 byte oracle query id with `oq_` prefix
+
+See the [identifier encoding scheme](./node/api/api_encoding.md) for the
```

Comment:
Broken link:
```suggestion
See the [identifier encoding scheme](/node/api/api_encoding.md) for the
```

I see this page names base58 though meaning Base58Check, and that is clarified in this link. Ok.